app/Http/Requests/UserRequest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UserRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|max:30',
+            'email' => 'required|email|max:255|unique:users,id'.$this->user_id,
```

Comment:
```suggestion
            'email' => 'required|email|max:255|unique:users,id' . $this->user_id,
```
convention sau và trước nối chuối có dấu cách